examples/github-complete/main.tf

```diff
@@ -0,0 +1,211 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+locals {
+  tags = {
+    Owner       = "user"
+    Environment = "dev"
+  }
+}
+
+##############################################################
+# Data sources for existing resources
+##############################################################
+
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+##############################################################
+# Atlantis Service
+##############################################################
+
+module "atlantis" {
+  source = "../../"
+
+  name = "atlantiscomplete"
+
+  # VPC
+  cidr            = "10.20.0.0/16"
+  azs             = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+  private_subnets = ["10.20.1.0/24", "10.20.2.0/24", "10.20.3.0/24"]
+  public_subnets  = ["10.20.101.0/24", "10.20.102.0/24", "10.20.103.0/24"]
+
+  # ECS
+  ecs_container_insights       = true
+  ecs_task_cpu                 = 512
+  ecs_task_memory              = 1024
+  container_memory_reservation = 256
+
+  entrypoint        = ["/bin/bash", "echo 'foo'", "docker-entrypoint.sh"]
```

Comment:
when i run this clean task in ecs fail to start with `/bin/bash: echo 'foo': No such file or directory`

Reply:
i tried commenting these but i still get errors:
`Error: initializing server: initializing terraform: running terraform version: : exit status` but i assume there are some missing params not being sent, not super familiar with atlantis.

Reply:
hmm, let me check this again as it worked when I was testing it out